### ISSUE TITLE
Gahz'Rilla punting too often

### ISSUE THREAD

**Issue by ghost** [Wall-core/Everlook-Bugtracker#439]
![image](https://user-images.githubusercontent.com/116027653/197708818-98d2384c-7ab9-45ef-9f6c-f3c1963da61a.png)
 
He punted the entire group every 2 seconds, this is not accurate and impossible to do. 
https://www.youtube.com/watch?v=yK5a4fGEE0g
I uploaded the issue to youtube, view this video^

**Comment by Simp-N** [comment]
Seems to be an issue that a lot of monsters spam their spells every chance they get.

**Comment by Tessablanc** [comment]
Yeah, that's what I was going to say. It seems like mob cooldowns in general are either missing or are absurdly short. See also the Web ability on the RFD gong spiders, which I reported yesterday.
